pynestml/grammars/PyNestMLParser.g4

```diff
@@ -54,303 +52,310 @@ parser grammar PyNestMLParser;
+                    | logicalOr=OR_KEYWORD;
 
   /**
-    ASTVariable Provides a 'marker' AST node to identify variables used in expressions.
-    @attribute name: The name of the variable without the differential order, e.g. V_m
-    @attribute vectorParameter: An optional array parameter, e.g., 'tau_syn ms[n_receptors]'.
-    @attribute differentialOrder: The corresponding differential order, e.g. 2
-  */
+   * ASTVariable Provides a 'marker' AST node to identify variables used in expressions.
+   * @attribute name: The name of the variable without the differential order, e.g. V_m
+   * @attribute vectorParameter: An optional array parameter, e.g., 'tau_syn ms[n_receptors]'.
+   * @attribute differentialOrder: The corresponding differential order, e.g. 2
+  **/
   variable : name=NAME
-  (LEFT_SQUARE_BRACKET vectorParameter=expression RIGHT_SQUARE_BRACKET)?
-  (DIFFERENTIAL_ORDER)*;
+             (LEFT_SQUARE_BRACKET vectorParameter=expression RIGHT_SQUARE_BRACKET)?
+             (DIFFERENTIAL_ORDER)*;
 
   /**
-    ASTFunctionCall Represents a function call, e.g. myFun("a", "b").
-    @attribute calleeName: The (qualified) name of the functions
-    @attribute args: Comma separated list of expressions representing parameters.
-  */
+   * ASTFunctionCall Represents a function call, e.g. myFun("a", "b").
+   * @attribute calleeName: The (qualified) name of the functions
+   * @attribute args: Comma separated list of expressions representing parameters.
+  **/
   functionCall : calleeName=NAME LEFT_PAREN (expression (COMMA expression)*)? RIGHT_PAREN;
 
-  /*********************************************************************************************************************
-  * Equations-Language
-  *********************************************************************************************************************/
+  // -------------------------------------------------------------------------
+  //  Equations
+  // -------------------------------------------------------------------------
 
   inlineExpression : (recordable=RECORDABLE_KEYWORD)? INLINE_KEYWORD variableName=NAME dataType EQUALS expression (SEMICOLON)? decorator=anyDecorator* NEWLINE;
 
   odeEquation : lhs=variable EQUALS rhs=expression (SEMICOLON)? decorator=anyDecorator* NEWLINE;
 
   kernel : KERNEL_KEYWORD variable EQUALS expression (KERNEL_JOINING variable EQUALS expression)* SEMICOLON? NEWLINE;
 
-  /*********************************************************************************************************************
-  * Procedural-Language
-  *********************************************************************************************************************/
+  // -------------------------------------------------------------------------
+  //  Procedural
+  // -------------------------------------------------------------------------
 
-  block : NEWLINE INDENT stmt+ DEDENT;
-
-  stmt : smallStmt | compoundStmt;
+  stmt : NEWLINE? (smallStmt | compoundStmt);
```

Comment:
Would this work if there were multiple newlines at the beginning of the `stmt`?

Reply:
I changed this so the newline is now in the ``stmtsBody`` rule. ``NEWLINE`` captures multiple newlines (this is already covered by the NESTML printer test).